### ISSUE TITLE
symbol after closing parenthesis will be included in url

### ISSUE THREAD

**Issue by vromero** [dkhamsing/awesome_bot#117]
The following markdown:

```
Unlike [SOAP](https://en.wikipedia.org/wiki/SOAP)-based
...
as [HTTP](https://en.wikipedia.org/wiki/HTTP), [URI](https://en.wikipedia.org/wiki/URI), [JSON](https://en.wikipedia.org/wiki/JSON), and [XML](https://en.wikipedia.org/wiki/XML). [[1](#references)]
```

Will trigger : 

```
Issues :-(
> Links 
  1. [L142] 404 https://en.wikipedia.org/wiki/JSON%2c  
  2. [L142] 404 https://en.wikipedia.org/wiki/SOAP-based  
  3. [L142] 404 https://en.wikipedia.org/wiki/URI%2c  
  4. [L142] 404 https://en.wikipedia.org/wiki/HTTP%2c  
```

**Comment by dkhamsing** [comment]
Thx
Will take a look

**Comment by dkhamsing** [comment]
@vromero thanks for the report
should be fixed with version 1.13.10

**Comment by vromero** [comment]
Thanks!
